sdk/python/feast/flags.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+FLAG_ALPHA_FEATURES_NAME = "ENABLE_ALPHA_FEATURES"
+FLAG_ON_DEMAND_TRANSFORM_NAME = "ENABLE_ON_DEMAND_TRANSFORMS"
+FLAG_PYTHON_FEATURE_SERVER_NAME = "ENABLE_PYTHON_FEATURE_SERVER"
```

Comment:
Do we need to have a per-functionality flag? What about having just one global flag?

Reply:
I think it could be nice to at least have a kill switch so users could use some but not all if there's some bad bug with one. Could make them enabled by default though?

Reply:
I'd definitely have them disabled by default. I am fine with the current approach. We can also consider using the `feature_store.yaml` for experimental config instead of env vars. I don't feel too strongly, but I think dealing with a bunch of env vars can get tiring beyond some point.

Reply:
Moved to feature_store.yaml + added CLI commands to modify